src/pages/viewpage/ViewPage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import TimeView from '../../components/time/TimeView'
+import Goal from '../../components/goal/Goal'
+import './viewpage.css'
+import Weather from '../../components/weather/Weather'
+import Quotes from '../../components/quotes/Quotes'
+import { GoogleSearch } from '../../components/google/GoogleSearch'
+import Stackoverflow from '../../components/stackoverflow/Stackoverflow'
```

Comment:
You can use shortcut alias here. It will make code cleaner (For reference: https://stackoverflow.com/questions/63067555/how-to-make-an-import-shortcut-alias-in-create-react-app)